app/assets/javascripts/search.js

```diff
@@ -23,4 +23,25 @@ jQuery(document).ready(function() {
     }
   })
 
+  $('#questions_searchform').submit(function(e){ 
+    e.preventDefault()
+    window.location = '/search/questions/'+$('#questions_searchform_input').val()
```

Comment:
@jywarren Yes, I noticed this new question action in the search controller.
@ananyo2012 if you can provide, required search option for your Q &A module from the search, I can facilitate more flexibility from my side. And yes, to avoid conflicts we can open new discussion for better understanding